src/display/text_layer.js

```diff
@@ -300,9 +300,16 @@ class TextLayer {
       angle += Math.PI / 2;
     }
 
-    const fontFamily =
+    let fontFamily =
       (this.#fontInspectorEnabled && style.fontSubstitution) ||
       style.fontFamily;
+
+    // Workaround for bug 1922063.
```

Comment:
Should we limit this to Firefox on Windows? While I haven't spotted any regressions for Linux, I'd feel better about landing this now (before the next release) if we can decrease the scope of this workaround as much as possible purely to the browser/OS listed in the bug.